### PR TITLE
[script.module.autocompletion@matrix] 2.0.6

### PR DESCRIPTION
**Pull request by finkleandeinhorn**
### Add-on details:

- **General**
  - Add-on name: AutoCompletion library
  - Add-on ID: script.module.autocompletion
  - Version number: 2.0.6
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/phil65/script.module.autocompletion
  
Module providing some AutoCompletion functions

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: script.module.autocompletion/.github/workflows/addon-submitter.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+name: Kodi Addon-Submitter
```
Comment (enen92) [inline script.module.autocompletion/.github/workflows/addon-submitter.yml:1]: Please drop, use the gitattributes to ignore this if using the submitter axtion

File: script.module.autocompletion/.gitignore [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Byte-compiled / optimized / DLL files
```
Comment (enen92) [inline script.module.autocompletion/.gitignore:1]: Same

File: script.module.autocompletion/addon.xml [hunk 3]
```diff
@@ -9,8 +9,8 @@
         <summary lang="en">Module providing some AutoCompletion functions</summary>
         <description lang="en">Module providing some AutoCompletion functions</description>
         <platform>all</platform>
-        <license>GPL-2.0-or-later</license>
-        <source>https://github.com/finkleandeinhorn/script.module.autocompletion</source>
+        <license>GNU GENERAL PUBLIC LICENSE Version 2.1, February 1999</license>
```
Comment (enen92) [inline script.module.autocompletion/addon.xml:12]: This was right before...

**Comment by finkleandeinhorn** [comment]
@enen92 Honestly not sure how those changes in `addon.xml` snuck through, but I've fixed them now, as well as dropping the files requested.

**Review by enen92 — APPROVED** [review]

**Pull request merged**